### DISCUSSION TITLE
Laravel - Oracle over yajra/laravel-oci8 failed

### DISCUSSION THREAD

**Discussion by ghost** [yajra/laravel-oci8#650] [Q&A]
I would like to connect a newly installed Laravel version (8.5.16) to an Oracle database via OCI8. For this I would use https://github.com/yajra/laravel-oci8. When I call the quick install (composer require yajra/laravel-oci8:^8), I get the following error message:

[InvalidArgumentException]
Package yajra/laravel-oci8 at version 8 has a PHP requirement incompatible with your PHP version, PHP extensions and Composer version

Currently I am using the...

...PHP version: 8.0.3 (cli)

...Composer version: 2.0.12

Would you have a tip what I can do about the error message?

**Comment by yajra** [comment]
Maybe try removing the constraint? `composer require yajra/laravel-oci8`

**Reply by ghost** [reply]
Thank you ⭐️⭐️⭐️⭐️⭐️
